Tests/Tests/STPTestingAPIClient.h

```diff
@@ -0,0 +1,28 @@
+//
+//  STPTestingAPIClient.h
+//  StripeiOS
+//
+//  Created by Cameron Sabol on 2/20/20.
+//  Copyright © 2020 Stripe, Inc. All rights reserved.
+//
+
+#import <Foundation/Foundation.h>
+
+#define STPTestingPublishableKey @"pk_test_ErsyMEOTudSjQR8hh0VrQr5X008sBXGOu6"
```

Comment:
why not `static const`?  Gives some type safety, at least. (I've been taught to avoid #defines when possible)

Reply:
ah yeah, I defined them to get around some compiler warning but I'll convert to static